docs/spec_draft.md

```diff
@@ -1576,6 +1578,48 @@ More formally, for each element `x`: `imag(x) = is_complex(x) ? x.imag : 0.0`.
 
 [Back to Ops](#index-of-ops)
 
+
+## stablehlo.infeed
+
+### Semantics
+
+Reads `results` from the infeed queue of the device. Multiple
+`stablehlo.infeed` operations are allowed in a computation, but there must be a
+total order among them.
```

Comment:
Reword suggestion for the second line - 
There must be a total order among all the `infeed` operations in a computation.

Since this looks like a constraint, we can move it to constraints section as well. WDYT?

Reply:
Two reason we should not view it as constraints:

1. All the properties are have mentioned in the constraints section are such that they can be verified at the local (op level). This property is not local. 
2. To me this property is like a recommendation for program correctness and termination and is common across other parallel programming domains. 

I put it as a recommendation only in the text.